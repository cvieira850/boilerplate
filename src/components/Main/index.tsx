const Main = () => (
  <main>
    <h1>React Avan√ßado - Boilerplate</h1>
  </main>
)

export default Main
