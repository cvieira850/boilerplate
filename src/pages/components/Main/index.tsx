const Main = () => (
  <main>
    <h1>Boilerplate do Caio</h1>
  </main>
)

export default Main
