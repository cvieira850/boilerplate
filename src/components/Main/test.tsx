import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  test('Should render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado - boilerplate/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
