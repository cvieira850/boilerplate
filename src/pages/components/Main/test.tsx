import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  test('Should render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /boilerplate do caio/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
