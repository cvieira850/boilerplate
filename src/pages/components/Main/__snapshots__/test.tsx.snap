// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should render the heading 1`] = `
<main>
  <h1>
    Boilerplate do Caio
  </h1>
</main>
`
